src/elements/hx-file-icon/index.spec.js

```diff
@@ -0,0 +1,154 @@
+import { fixture, expect } from '@open-wc/testing';
+
+/**
+ * <hx-file-icon> component tests
+ *
+ * @type HXFileIconElement
+ *
+ */
+describe('<hx-file-icon> component tests', () => {
+    const template = '<hx-file-icon>';
+
+    describe('test instantiate element', () => {
+        it('should be instantiated with hx-defined attribute', async () => {
+            const component = /** @type {HXFileIconElement} */ await fixture(template);
+            const attr = component.hasAttribute('hx-defined');
+
+            expect(attr).to.be.true;
+        });
+
+        it('should not be hidden', async () => {
+            const component = /** @type {HXFileIconElement} */ await fixture(template);
+            const prop = component.hidden;
+
+            expect(prop).to.be.false;
+        });
+
+        it(`the rendered Light DOM should NOT equal simple template ${template}`, async () => {
+            const component = /** @type {HXFileIconElement} */ await fixture(template);
+
+            expect(component).lightDom.to.not.equal(template);
+        });
+
+    });
+
+    describe('test Shadow DOM', () => {
+        describe('verify render', () => {
+            it('should have a static Shadow DOM', async function () {
+                const component = /** @type { HXFileIconElement } */ await fixture(template);
+
+                // FIXME: temp fix issue submitted https://github.com/open-wc/open-wc/issues/1540
+                const shadow = component.shadowRoot.innerHTML;
+
+                expect(component).shadowDom.to.equal(shadow);
+            });
+
+            it('should render the Shadow Root mode open', async () => {
+                const component = /** @type { HXFileIconElement } */ await fixture(template);
+                const mode = component.shadowRoot.mode;
+
+                expect(mode).to.equal("open");
+            });
+
+            it('should have a single <slot>', async () => {
+                const component = /** @type { HXFileIconElement } */ await fixture(template);
+                const shadow = component.shadowRoot;
+                const query = shadow.querySelectorAll('slot');
+                const len = query.length;
+
+                expect(len).to.be.equal(0);
+            });
```

Comment:
Please check this test.  Is there a single `slot` or no slot?

Reply:
As we are not using the slot in shadow.html file, So I completely removed slot section.